cocos/asset/asset-manager/asset-manager.ts

```diff
@@ -109,85 +104,90 @@ export interface IAssetManagerOptions {
 /**
  * @en
  * This module controls asset's behaviors and information, include loading, releasing etc. it is a singleton
- * All member can be accessed with `assetManager`.
+ * You can access it via [[assetManager]].
  *
  * @zh
- * 此模块管理资源的行为和信息，包括加载，释放等，这是一个单例，所有成员能够通过 `assetManager` 调用
- *
+ * 此模块管理资源的行为和信息，包括加载，释放等，这是一个单例，你能够通过 `assetManager` 访问它。
  */
 export class AssetManager {
     /**
      * @en
-     * Normal loading pipeline
+     * Normal loading pipeline. Asset manager uses this pipeline to load all assets, it has an additional asset parsing process compared to the fetchPipeline.
      *
      * @zh
-     * 正常加载管线
-     *
+     * 正常加载管线。Asset manager 使用此管线来加载所有资源，他与 fetchPipeline 相比额外多了资源的解析过程。
      */
     public pipeline: Pipeline = pipeline.append(preprocess).append(load);
 
     /**
      * @en
-     * Fetching pipeline
+     * Fetching pipeline. Asset manager uses this pipeline to preload all assets, which lacks the asset parsing process compared to the pipeline.
      *
      * @zh
-     * 下载管线
+     * 下载管线。Asset manager 使用此管线来预加载所有资源，他与 pipeline 相比缺少了资源的解析过程。
      *
      */
     public fetchPipeline: Pipeline = fetchPipeline.append(preprocess).append(fetch);
 
     /**
      * @en
-     * Url transformer
+     * Url transformer. Asset manager uses this pipeline to convert the uuid, path, etc. to the path of the final file path to be loaded.
+     * You can customize this pipeline to redirect the path to the path you want.
      *
      * @zh
-     * Url 转换器
+     * Url 转换器。Asset manager 使用此管线将 uuid, 路径等信息转换为最终要加载的文件路径。你可以自定义此管线将路径重定向到你想要的路径。
      *
      */
     public transformPipeline: Pipeline = transformPipeline.append(parse).append(replaceOverrideAsset).append(combine);
 
     /**
      * @en
-     * The collection of bundle which is already loaded, you can remove cache with [[removeBundle]]
+     * The collection of bundle which is already loaded, you can remove cache with [[removeBundle]].
      *
      * @zh
-     * 已加载 bundle 的集合， 你能通过 [[removeBundle]] 来移除缓存
+     * 已加载 bundle 的集合， 你能通过 [[removeBundle]] 来移除缓存。
      *
      */
     public bundles: ICache<Bundle> = bundles;
 
     /**
      * @en
-     * The collection of asset which is already loaded, you can remove cache with [[releaseAsset]]
+     * The collection of asset which is already loaded, you can remove cache with [[releaseAsset]].
      *
      * @zh
-     * 已加载资源的集合， 你能通过 [[releaseAsset]] 来移除缓存
+     * 已加载资源的集合， 你能通过 [[releaseAsset]] 来移除缓存。
      */
     public assets: ICache<Asset> = assets;
 
     /**
-     * @internal only using in L10N for now
+     * @internal only using in L10N for now.
      */
     public readonly assetsOverrideMap = assetsOverrideMap;
 
+    /**
+     * @internal
+     */
     public generalImportBase = '';
+    /**
+     * @internal
+     */
```

Comment:
Does `@internal` work? I think it will still export the variable.

Reply:
Yeah, editor use this api, so I can not use `@engineInternal`. I think maybe add another comment likes `@editor` to solve this problem.